covid_xprize/examples/predictors/conditional_lstm/train_predictor.py

```diff
@@ -149,24 +149,22 @@ def train_predictor(training_data: pd.DataFrame,
     # Gather test info
     country_indeps = []
     country_predss = []
-    country_casess = []
     for model in models:
-        country_indep, country_preds, country_cases = _lstm_get_test_rollouts(model,
-                                                                              df,
-                                                                              test_countries,
-                                                                              country_samples,
-                                                                              context_column)
+        country_indep, country_preds = _lstm_get_test_rollouts( model,
+                                                                df,
+                                                                test_countries,
+                                                                country_samples,
+                                                                context_column)
         country_indeps.append(country_indep)
         country_predss.append(country_preds)
-        country_casess.append(country_cases)
 
     # Compute daily smooth cases per 100K mae
     test_case_maes = []
     for m in range(len(models)):
         total_loss = 0.
         for c in test_countries:
             true_cases = np.array(df[df.GeoID == c].SmoothNewCasesPer100K)[-nb_test_days:]
-            pred_cases = country_casess[m][c][-nb_test_days:]
+            pred_cases = country_predss[m][c][-nb_test_days:]
```

Comment:
That was a bug?

Reply:
Sort of -- the code as it was had `country_predss == country_casess`, so it made no difference. It was dead code, a holdover from when we were comparing on the actual cases column and not the SmoothNewCasesPer100K context column.